src/transformers/models/bert/modeling_bert.py

```diff
@@ -692,6 +807,10 @@ def __init__(self, config):
         # Need a link between the two variables so that the bias is correctly resized with `resize_token_embeddings`
         self.decoder.bias = self.bias
 
+    def _tie_weights(self):
```

Comment:
This fix was added due to a [test failure](https://gist.github.com/hackyon/cd251c705aadb2bc77ffffcf0233c0ce) that uncovered an existing bug.

The head was initialized but the weights weren't retied as necessary. This was causing `self.decoder.bias` to be different from `self.bias`. When loading the pretrained model with `low_cpu_mem_usage=True`, the `self.decoder.bias` had uninitiated params (with `device=meta`) whereas `self.bias` was set properly (with `device=cpu`)

I'm slightly concerned this will affect the output some users see when using this model. Please let me know what you think about this.

Reply:
I pulled this out to its own PR here:
https://github.com/huggingface/transformers/pull/28948

This issue is unrelated to SDPA, but was just uncovered by a SPDA test, so I just pulled it out to its own PR.

Reply:
Addition looks OK to me - thanks for digging into this. 

> I'm slightly concerned this will affect the output some users see when using this model. Please let me know what you think about this.

Could you expand on what you think might be an issue?

Reply:
I was initially concerned that users were loading and using the model with a wrong bias (ie. device=meta), and this fix to use the correct bias will cause the results to change between versions.

However, but that seems unlikely after playing around with this a bit more - turns out it was quite difficult to run the model when the bias had device=meta, so I doubt anyone was actually running the model in this particular configuration before the fix.